src/commands/dev/mod.rs

```diff
@@ -65,6 +74,22 @@ pub async fn dev(
     Ok(())
 }
 
+fn munge_response_headers(parts: &mut Parts) -> Result<(), failure::Error> {
```

Comment:
nit: could we use a term more specific than munge, like `strip_cf_prefix_from_headers`?